--- circonus-napp-0.1r6841/opt/napp/bin/issue-refresh.orig	2010-12-06 15:08:08.000000000 +0000
+++ circonus-napp-0.1r6841/opt/napp/bin/issue-refresh	2010-12-22 18:12:04.000000000 +0000
@@ -1,4 +1,13 @@
-#!/bin/sh
+#!/bin/bash
+#
+# issue-refresh 
+#
+# chkconfig: 345 11 89
+# description: first run of issue-refresh should be immediately after network up
+#	       and before the console opens
+# processname: ip
+# config: /etc/issue
+
 IP=`/sbin/ip addr show dev eth0 2>&1 | /bin/awk '{if($1=="inet"){print $2;}}'`
 if [ -z "$IP" ]; then
 	IP="(no address assigned)"
